tests/generic_config_updater/test_eth_interface.py

```diff
@@ -320,6 +320,7 @@ def test_update_valid_index(duthosts, rand_one_dut_hostname, ensure_dut_readines
         delete_tmpfile(duthost, tmpfile)
 
 
+@pytest.mark.skip(reason="Bypass as this is not a production scenario")
```

Comment:
In L2 mode, there is step to config speed, so it is production scenario [L2 Switch mode · sonic-net/SONiC Wiki (github.com)](https://github.com/sonic-net/SONiC/wiki/L2-Switch-mode)